### DISCUSSION TITLE
Site slower with Redis enabled

### DISCUSSION THREAD

**Discussion by Dan60011** [rhubarbgroup/.github#9] [Support (ROC)] [answered]
Please see below

Status: Connected
Client: Predis (v2.1.2)
Drop-in: Valid
Disabled: No
Ping: PONG
Errors: []
PhpRedis: Not loaded
Relay: Not loaded
Predis: 2.1.2
Credis: Not loaded
PHP Version: 8.1.19
Plugin Version: 2.4.1
Redis Version: 3.2.12
Multisite: No
Metrics: Enabled
Metrics recorded: 17
Filesystem: Working
Global Prefix: "wp_"
Blog Prefix: "wp_"
WP_REDIS_HOST: "127.0.0.1"
WP_REDIS_PORT: 6379
WP_REDIS_DATABASE: 1
WP_REDIS_TIMEOUT: 1
WP_REDIS_READ_TIMEOUT: 1
WP_REDIS_PREFIX: "coxmotorparts_digitaloft_wp4_"
WP_CACHE_KEY_SALT: "coxmotorparts_digitaloft_wp4_"
WP_REDIS_PLUGIN_PATH: "/home/coxmotorparts/public_html/wp-content/plugins/redis-cache"
Global Groups: [
    "blog-details",
    "blog-id-cache",
    "blog-lookup",
    "global-posts",
    "networks",
    "rss",
    "sites",
    "site-details",
    "site-lookup",
    "site-options",
    "site-transient",
    "users",
    "useremail",
    "userlogins",
    "usermeta",
    "user_meta",
    "userslugs",
    "redis-cache",
    "blog_meta"
]
Ignored Groups: [
    "counts",
    "plugins",
    "themes"
]
Unflushable Groups: []
Groups Types: {
    "blog-details": "global",
    "blog-id-cache": "global",
    "blog-lookup": "global",
    "global-posts": "global",
    "networks": "global",
    "rss": "global",
    "sites": "global",
    "site-details": "global",
    "site-lookup": "global",
    "site-options": "global",
    "site-transient": "global",
    "users": "global",
    "useremail": "global",
    "userlogins": "global",
    "usermeta": "global",
    "user_meta": "global",
    "userslugs": "global",
    "redis-cache": "global",
    "counts": "ignored",
    "plugins": "ignored",
    "themes": "ignored",
    "blog_meta": "global"
}
Drop-ins: [
    "advanced-cache.php v by ",
    "maintenance.php v by ",
    "Redis Object Cache Drop-In v2.4.1 by Till Krüss"
]

**Comment by lkraav** [comment]
> Site slower

...as measured by what?

**Reply by Dan60011** [reply]
Query Monitor

**Comment by tillkruss** [comment, marked as answer]
There is not nearly enough information to go on to provide any kind of support.
